rllib/utils/space_utils.py

```diff
@@ -0,0 +1,53 @@
+from gym.spaces import Tuple, Dict
+
+
+def flatten_space(space):
```

Comment:
Don't need this any more.

Reply:
Yeah, we do, unfortunately. gym.Spaces are not flattened by tree b/c a gym.Dict is not an actual dict, gym.Tuple is not an actual tuple.

Reply:
Got it. I guess this is currently done implicitly by the dict/tuple flattening preprocessors for observations.